config/locales/en/helpers.yml

```diff
@@ -241,13 +239,9 @@ en:
           adult_sexual_harm_prevention_order: A court ruled that you pose a risk of causing a sexual offence or sexual harm
           # adult_motoring
           adult_disqualification: You were banned from driving
-          adult_motoring_fine: A court gave you a fine that was not a fixed penalty notice (FPN)
```

Comment:
Beware, I think `adult_motoring_fine` was removed by mistake. It belongs to motoring fines that we still want in the service.